ansible/roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,136 @@
+---
+
+- name: copy filebeat ILM policy
+  template:
+    src: 'ilm_policy.json.j2'
```

Comment:
If you do not use templates you can put to `files` fir instead of `templates` and use https://docs.ansible.com/ansible/2.5/modules/copy_module.html